xwiki-platform-core/xwiki-platform-blog/xwiki-platform-blog-ui/src/main/resources/Blog/Categories.xml

```diff
@@ -1,5 +1,4 @@
 <?xml version="1.0" encoding="UTF-8"?>
-
```

Comment:
You need to run `mvn xar:format` twice because of some bug. If you do so, this blank line will remain.